docker-compose-native/prestissimo/workers/1/config.properties

```diff
@@ -2,3 +2,4 @@ coordinator=false
 http-server.http.port=8080
 discovery.uri=http://172.19.0.4:8080
```

Comment:
Can you use the service name?

Reply:
@wanglinsong I'm not sure what you mean

Reply:
Use service name instead of ip address to reference coordinator.

Reply:
Like in https://github.ibm.com/lakehouse/presto-performance/blob/main/presto-deploy-cluster/clusters/large/workers/config.properties